agasc/tests/test_agasc_2.py

```diff
@@ -581,3 +581,44 @@ def test_get_supplement_table_obs():
 def test_get_supplement_table_obs_dict():
     obs = agasc.get_supplement_table("obs", as_dict=True)
     assert isinstance(obs, dict)
+
+
+def test_write():
+    import tempfile
+    from pathlib import Path
+
+    import tables
+
+    from agasc import write_agasc
+
+    with (
+        tempfile.TemporaryDirectory() as tempdir,
+        tables.open_file(
+            Path(os.environ["SKA"]) / "data" / "agasc" / "agasc_1p7.h5"
+        ) as h5_in,
+    ):
+        t = Table(h5_in.root.data[:1000])
```

Comment:
This is the only line that needs the `h5_in` context, so out-dent all the rest to take them out of the context manager.

Reply:
Done